lib/core/src/lib/clipboard/clipboard.directive.spec.ts

```diff
@@ -128,6 +137,16 @@ describe('CopyClipboardDirective', () => {
         expect(navigator.clipboard.writeText).toHaveBeenCalledWith('text to copy');
     }));
 
+    it('should copy the content of element on mouse enter', fakeAsync(() => {
```

Comment:
I think title is incorrect. It says: "mouse enter" but your event is different.

Reply:
title changed and added keydown instead of mouse enter.